.github/workflows/conventional-commit-pr-title-reminder.yml

```diff
@@ -0,0 +1,57 @@
+name: Conventional commit PR Title Reminder
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - enterprise/frontend/src/embedding-sdk/**
+
+jobs:
+  pr-title-reminder:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Check if the PR title is already following Conventional Commits style
+        uses: actions/github-script@v7
+        id: is-conventional-commit
+        with:
+          result-encoding: string
+          script: | # js
```

Comment:
TIL: `# js` highlights code in the editor.
![image](https://github.com/metabase/metabase/assets/1937582/70800729-9249-4792-896d-fc7ae9a05639)